Sources/Apodini/Environment/Environment.swift

```diff
@@ -0,0 +1,60 @@
+/// A key for accessing values in the environment.
+public protocol EnvironmentKey {
+    /// Represents the type of the environment key’s value.
+    associatedtype Value
+    /// The default value of the `EnvironmentKey`.
+    static var defaultValue: Self.Value { get }
+}
+
+/// A collection of environment values.
+/// Custom environment values can be created by extending this struct with new properties.
+public struct EnvironmentValues {
+    var values: [ObjectIdentifier: Any] = [:]
+    
+    /// Initializer of `EnvironmentValues`.
+    public init() { }
```

Comment:
I would suggest to make this initializer internal as we should only create one instance of an `EnvironmentValues` per web service.